std/container/slist.d

```diff
@@ -140,7 +140,8 @@ Constructor taking a number of nodes
      */
     this(U)(U[] values...) if (isImplicitlyConvertible!(U, T))
     {
-        insertFront(values);
+        U[] valuesDuped = values.dup;
```

Comment:
This is painful and it's inane that we have to do this for DIP1000. Would manually inlining the `insertFront` loop remove the need to copy the array?

Reply:
What exactly is meant by `manually inlining the insertFront loop`?
I tried foreach lowered to   for (auto item=stuff.front...
That didn't change the error: Error: scope variable item may not be returned